### DISCUSSION TITLE
CPC200-Autokit

### DISCUSSION THREAD

**Discussion by Ar4erR07** [ludwig-v/wireless-carplay-dongle-reverse-engineering#116] [Show and tell]
Hello. I have device as in photo. I made working dump, maybe it will come in handy for someone. I wonder if some modified firmware fits me from this repository.
![0FB1CB01-31BB-43DF-9F92-0764022E0E34](https://user-images.githubusercontent.com/16082311/135145622-fc817495-4349-419d-a513-4a60c8fbac09.jpeg)
![D4C62FB7-CC22-487C-B8C0-D6F42F46BAC8](https://user-images.githubusercontent.com/16082311/135145642-0ab79637-e0ba-4ec0-8a09-bd7fc7fc3d77.jpeg)
![D494D27F-EEE4-4F4D-AB8D-0208D2C08032](https://user-images.githubusercontent.com/16082311/135145652-78433f2f-6dec-43d4-9075-ed31d390ced1.jpeg)
![F43B4B9B-E834-4C11-9F6D-D79A40F0A919](https://user-images.githubusercontent.com/16082311/135145655-c472b493-c6b8-4d8c-bb09-bc47d529c329.jpeg)
![image](https://user-images.githubusercontent.com/16082311/135145659-a9e2f9a5-9339-47b3-a27d-971bbbb66960.jpg)
[Flash dump](https://drive.google.com/file/d/1xuz8dKB-BbpCzlK3qL4zZSl7dbNm3Msn/view?usp=sharing)

**Comment by ludwig-v** [comment]
@jsm174 would be interested :)

**Comment by ludwig-v** [comment]
You can't convert it because your device will be missing the MFI chip to work as Carplay proxy

**Reply by Ar4erR07** [reply]
Thanks for the answer. I don't want to convert it. I want to either find or make firmware with SSH and improvements. Firmware version in the dump 2021.08.30.1821

**Comment by Ar4erR07** [comment]
I pulled out two update images for my device. Can you give advice on how to unpack them? I understand that they are encrypted using aes. And the decryption is done by ARMimg_maker on the device

**Comment by Ar4erR07** [comment]
[crootfs.tar.gz](https://github.com/ludwig-v/wireless-carplay-dongle-reverse-engineering/files/8603504/crootfs.tar.gz)
[moddump.bin.zip](https://github.com/ludwig-v/wireless-carplay-dongle-reverse-engineering/files/8603518/moddump.bin.zip)
Not tested custom modded dump for Android Based Autokit dongle, firmware version 2021.12.22.1647. crootfs.tar.gz -copy from roots to create jffs2 root. In firmware enabled dropbear, wpa2, load script from usb flash drive, changed wifi password, enabled performance governor. Based on @qdk0901 version
![Снимок экрана 2022-05-02 в 18 43 32](https://user-images.githubusercontent.com/16082311/166254081-2183ea11-775b-4e9e-ac02-20eb78d92748.png)

**Reply by qba667** [reply]
@Ar4erR07 can you please provide output of  cat /proc/cpuinfo?
I am wondering if this is same CPU as other unit CPU.
